keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -198,17 +198,19 @@ If CSI driver does not have `GET_VOLUME` controller capability(or `ControllerGet
 
 * **Can the feature be disabled once it has been enabled (i.e. can we rollback
   the enablement)?**
-  Yes the feature gate can be disabled once enabled. The quota code will use new field to calculate quota if available. This will allow quota code to use
-  `pvc.Status.AllocatedResources` even if feature gate is disabled.
+  Yes the feature gate can be disabled once enabled. However quota resources present in the cluster will be based off a field(`pvc.Status.AllocatedResources`) which is no longer updated.
```

Comment:
If the calculation of the quota controller and admission will be stale, I think the answer here is "No, disabling the feature prevents resizing, but the quota calculation can be stale and stuck.  <and the other details you have now>".  When feature disablement leaves clusters stuck in in-between states, we need to call it out clearly here.

If you have steps that can be run to put the cluster back to a pre-enablement state, you can list them here and keep the answer as yes.  We do this for kubelet features which can require evicting workloads to reset.

Reply:
I have not tested it. But I think if you delete and re-create previously created `ResouceQuota` objects, it will force quota controller to recalculate quota entirely based off `pvc.Spec.Resources` so that workaround works. I will mention that.

Reply:
updated with the workaround.

Reply:
When the feature is disabled, wouldn't quota controller go back to the previous logic of only looking at `pvc.Spec.Resources`?

Reply:
The quota is calculated and stored in `ResourceQuota`'s status and do not automatically get updated unless something causes their recalculation. So yeah stored quota might be stale.